### DISCUSSION TITLE
cannot import name 'models' from 'keras_nlp'

### DISCUSSION THREAD

**Discussion by Chocolate0086** [keras-team/keras-hub#1250] [Q&A]
When keras_nlp is installed successfully, I cannot import name 'models' from 'keras_nlp'. No 'models' can be found in the installation path. Can anyone tell me how to fix it?

**Comment by jbischof** [comment]
@Chocolate0086 could you please create an issue (not a discussion) and attach a colab reproducing the error you're getting? Thanks!

The discussion tab and is only well suited for high level discussions, not bug triage.
